Add missing xcb install

gillamkid opened the pull request:
I freshly installed Ubuntu LTS 20.4 and followed the `Download and Install` instructions. The app image failed to run because I was missing the libxcb-xinerama0 package. The solution to the error was not immediately obvious and many Ubuntu users will likely face the same problem unless the `Download and install` instructions have the user install the missing package.

Here is the original error I got.

```sh
sam@hp:~/Downloads$ ./QGroundControl.AppImage 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, xcb.

Aborted (core dumped)
```

hamishwillee commented on en/getting_started/download_and_install.md:
```diff
@@ -53,7 +53,7 @@ Before installing *QGroundControl* for the first time:
    sudo usermod -a -G dialout $USER
    sudo apt-get remove modemmanager -y
-   sudo apt install gstreamer1.0-plugins-bad gstreamer1.0-libav gstreamer1.0-gl -y
+   sudo apt install gstreamer1.0-plugins-bad gstreamer1.0-libav gstreamer1.0-gl libxcb-xinerama0 -y
```
@DonLakeFlyer This looks reasonable, but is news to me. When did the dependency go in?

gillamkid replied:
Only the most recent release (4.2.0) needs the package, the previous stable version (4.1.4) does not.

hamishwillee reviewed (changes requested):
Thanks for the fix. Can you please undo all the changes except for the one in the /en path. 

The way our translation system (crowdin) works, the english version goes in and the translations are updated automatically (e.g. /ru /ko etc.. So if I merged this when crowdin did the update I would need to fix up conflicts.

Commit:
Add missing libxcb-xinerama0 install

If the user starts with newly installed Ubuntu LTS 20.04, the
libxcb-xinerama0 library needs to be installed or the AppImage
will fail to run

DonLakeFlyer commented:
Shouldn't the AppImage creation be updated to include this in the package? That should just always work out of the box.

hamishwillee commented:
I'm guessing this is invalid. If not, I migrated docs to https://github.com/mavlink/qgroundcontrol/tree/master/docs/en

Can you copy the PR across? I will be archiving this in a week or so.
